sdks/wasm/src/driver.c

```diff
@@ -674,3 +680,146 @@ mono_wasm_enable_on_demand_gc (void)
 {
 	mono_wasm_enable_gc = 1;
 }
+
```

Comment:
This should follow the existing coding conventions in this file.

Reply:
Fixed in commit